packages/nodes-base/nodes/ExecuteWorkflowTrigger/ExecuteWorkflowTrigger.node.ts

```diff
@@ -46,10 +51,129 @@ export class ExecuteWorkflowTrigger implements INodeType {
 				],
 				default: 'worklfow_call',
 			},
+			{
+				displayName: 'Input Fields',
+				name: FIELDS,
+				placeholder: 'Add Field',
+				type: 'fixedCollection',
+				description:
+					'Define expected input fields. If no inputs are provided, all data from the calling workflow will be passed through.',
+				typeOptions: {
+					multipleValues: true,
+					sortable: true,
+				},
+				displayOptions: {
+					show: { '@version': [{ _cnd: { gte: 1.1 } }] },
+				},
+				default: {},
+				options: [
+					{
+						name: VALUES,
+						displayName: 'Values',
+						values: [
+							{
+								displayName: 'Name',
+								name: 'name',
+								type: 'string',
+								default: '',
+								placeholder: 'e.g. fieldName',
+								description: 'Name of the field',
+							},
+							{
+								displayName: 'Type',
```

Comment:
For this phase we won't be using types, [just names](https://www.notion.so/n8n/Define-input-for-sub-workflows-12a5b6e0c94f804f9aa1fb118399b008?pvs=4#12f5b6e0c94f807b91fcd4617858dd0d), so we don't need to add UI from them.

Reply:
I'll probably just do the types now if you don't mind. I'm basing this on the Set node which has types, so it'd be more work to fully drop them now just to readd. We definitely want them before merging to master, right? :o